lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -144,6 +154,51 @@ bool PatchInOutImportExport::runOnModule(Module &module) {
   return true;
 }
 
+// =====================================================================================================================
+// Mark the 'done' flag to the very last position export instruction.
+//
+// @param [in/out] func : LLVM function to be run on
+// @param [in] pdt : The PostDominatorTree of the \p func
```

Comment:
pdt -> postDomTree

According to the coding convention, we avoid use acronym-style abbreviation. 

Also, you can avoid "[in]" tag. We assume all parameters without extra explanations are input, so we only add [out] or [in/out] tag for output and inout parameters.